web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -866,4 +866,29 @@ describe("buildUserRoadmap", () => {
       expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toEqual([]);
     });
   });
+
+  describe("raffle-bingo games business", () => {
+    it("adds raffle-bingo-games task if raffle-bingo-games business", () => {
+      const profileData = generateStartingProfile({
+        raffleBingoGames: true,
+        isNonprofitOnboardingRadio: true,
+        legalStructureId: "nonprofit",
+      });
+
+      buildUserRoadmap(profileData);
+
+      expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toContain("raffle-bingo-games");
+    });
+
+    it("does not add raffle-bingo-games task if not raffle-bingo-games business", () => {
+      buildUserRoadmap(
+        generateStartingProfile({
+          raffleBingoGames: false,
+          isNonprofitOnboardingRadio: false,
+          legalStructureId: "limited-liability-company",
+        })
+      );
+      expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).not.toContain("raffle-bingo-games");
+    });
+  });
```

Comment:
[boulder] We should be covering all our negative test cases here as well. Given that we are fully expecting that a business _must_ be a nonprofit in order to get this addon, there should be a test for the case that `raffleBingoGames` is `true`, `isNonprofitOnboardingRadio` is `false` and we assert that the addon does not get added.